src/libsyntax/feature_gate.rs

```diff
@@ -378,6 +374,8 @@ declare_features! (
     // Allows `..` in tuple (struct) patterns
     (accepted, dotdot_in_tuple_patterns, "1.14.0", Some(33627)),
     (accepted, item_like_imports, "1.14.0", Some(35120)),
+    // Allows using `Self` and associated types in struct expressions and patterns.
+    (accepted, more_struct_aliases, "1.14.0", Some(37544)),
```

Comment:
> What does this version mean exactly for accepted and removed features, something else?

Note [the comment](https://github.com/rust-lang/rust/blob/d7777ae6/src/libsyntax/feature_gate.rs#L96-L97) above the first `declare_features!` block: "The version numbers here correspond to the version in which the current status was set."  

> I guess here it doesn't actually show anywhere?

It is at least [displayed in the message of the lint](https://github.com/rust-lang/rust/pull/33973) for `#[feature]`-gate attributes for already-stable features.